onnxruntime/core/common/utf8_util.h

```diff
@@ -0,0 +1,129 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "core/common/common.h"
+
+namespace onnxruntime {
+namespace utf8_util {
+
+// Returns the number of bytes in the utf8 character
+// by analyzing its leading byte
+inline bool utf8_bytes(unsigned char ch, size_t& len) {
```

Comment:
Can we add unit tests for the functions in this file?

Reply:
It is added though tested indirectly but sufficiently

---
In reply to: [238894458](https://github.com/Microsoft/onnxruntime/pull/31#discussion_r238894458) [](ancestors = 238894458)